README.md

```diff
@@ -35,28 +35,37 @@ not present any other legal issues.
 * OSI approved license are strongly preferred.
 * Mods do not need to be minetest-licensing compatible (meaning, (A)GPL3.0+ is fine).
 * Non-OSI licenses may be acceptable, but this is determined on a one-by-one case.
-* Pure Public Domain is not acceptable, however WTFPL, MIT or similar usually is acceptable.
+* Pure Public Domain is not acceptable, however ISC, MIT or similar usually is acceptable.
```

Comment:
what about `CC0`, does that count as "pure public domain"?

Reply:
No, I consider it not the same. For now, CC0 is, in my opinion, OK for the time being.